core/src/scss/components/hero/_hero--caption.scss

```diff
@@ -31,15 +31,24 @@
     @include modular-spacing(padding-bottom, 2);
     text-align: right;
```

Comment:
This might be a @kerri-augenstein question. When the caption container isn't full width, it looks good with `text-align: right`, but I think it looks a bit unnatural at xs to md breakpoints when the caption takes up the full width and the text is still align to the right. Could we make the caption left-aligned or center-aligned at xs to md breakpoints?